packages/components/src/form-token-field/style.scss

```diff
@@ -181,4 +181,15 @@
 		background: $components-color-accent;
 		color: $white;
 	}
+
+	&.is-disabled,
+	&[aria-disabled="true"] {
+		opacity: 0.25;
```

Comment:
I think we want an explicit `color` here (`$gray-600`, I think) rather than an opacity, to ensure that the text meets the contrast threshold.

Reply:
I followed other implementations. Let's try it

Reply:
https://github.com/WordPress/gutenberg/pull/61294/commits/87290d852245cf3fa1390d060841744648c875d5